contrib/oath-toolkit/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "oath-toolkit"
+pkgver = "2.6.11"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+make_dir = "."
```

Comment:
@nekopsykose ^ this bit fixes the `./gdoc` issue. Building out of tree isn't supported it seems

Reply:
nice :)